frontend/public/components/resource-pages.ts

```diff
@@ -2,13 +2,13 @@
 
 import { Map as ImmutableMap } from 'immutable';
 
-import { ReportReference, ReportGenerationQueryReference } from './chargeback';
```

Comment:
Had to remove this one because of issue with circular deps.
Haven't come up with better solution then to remove this `import` and reinitialize `ReportReference` and `ReportGenerationQueryReference`.